### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by IRIKA**

### CODE REVIEW

File: GameStore/Game_Store.DAL/Interfaces/IGenericRepository.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace GameStore.DAL.Interfaces
+{
+    public interface IGenericRepository<TEntity> where TEntity : class
```
Comment (IrynaShemet) [inline GameStore/Game_Store.DAL/Interfaces/IGenericRepository.cs:7]: Add IEntityWithId after the class

File: GameStore/Game_Store.WEB/Controllers/GameController.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        public ActionResult GetByPlatformTypes(List<int> platformTypes)
+        {
+            return Ok(_gameService.GetGamesByPlatformTypes(platformTypes));
+        }
+
+        [HttpGet]
+        [Route("/game/{gamekey}/download")] 
+        public ActionResult DownloadGame(string gamekey)
+        {
+            var path = @"C:/Temp/download.txt";
```
Comment (IrynaShemet) [inline GameStore/Game_Store.WEB/Controllers/GameController.cs:118]: Use relative path

File: GameStore/Game_Store.WEB/Startup.cs [hunk 3]
```diff
@@ -23,22 +22,38 @@ public Startup(IConfiguration configuration)
...
             }
-            else
-            {
-                app.UseExceptionHandler("/Home/Error");
-                // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
-                app.UseHsts();
-            }
+            //else
+            //{
+            //    // ��������� ������ HTTP
```
Comment (IrynaShemet) [inline GameStore/Game_Store.WEB/Startup.cs:50]: Translate

File: GameStore/Game_Store.DAL/Interfaces/IGenericRepository.cs (hunk 1 above)
Comment (IRIKA) [reply GameStore/Game_Store.DAL/Interfaces/IGenericRepository.cs:7]: I have added interface but didn't understand what for.
I used this interface at class EFGenericRepository.
Have I to refactor my code at IGenericRepository&

File: GameStore/Game_Store.WEB/Controllers/GameController.cs (hunk 2 above)
Comment (IRIKA) [reply GameStore/Game_Store.WEB/Controllers/GameController.cs:118]: It has fixed

**Pull request merged**